### ISSUE TITLE
KeyError: 'num' occurs while tokenization.

### ISSUE THREAD

**Issue by dwarfer7634** [megagonlabs/ginza#39]
```
import spacy
nlp = spacy.load('ja_ginza_nopn')

doc = nlp('2016.9 - 2017.2')

for token in doc:
    print(token.text)
```

I'm using Ginza to split Japanse sentences into tokens.
And, the error below occured.
The snippet above is the minimal code which reproduces the error I bumped into.
Does anyone know the way to solve this problem? 

```
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-23-9a0d074dcdf0> in <module>
      2 nlp = spacy.load('ja_ginza_nopn')
      3 
----> 4 doc = nlp('2016.9 - 2017.2')
      5 
      6 for token in doc:

~/miniconda3/envs/shinra/lib/python3.6/site-packages/spacy/language.py in __call__(self, text, disable, component_cfg)
    383             if not hasattr(proc, "__call__"):
    384                 raise ValueError(Errors.E003.format(component=type(proc), name=name))
--> 385             doc = proc(doc, **component_cfg.get(name, {}))
    386             if doc is None:
    387                 raise ValueError(Errors.E005.format(name=name))

~/miniconda3/envs/shinra/lib/python3.6/site-packages/ginza/japanese_corrector.py in __call__(self, doc)
     18 
     19     def __call__(self, doc):
---> 20         correct_dep(doc)
     21         set_bunsetu_bi_type(doc)
     22         return doc

~/miniconda3/envs/shinra/lib/python3.6/site-packages/ginza/japanese_corrector.py in correct_dep(doc)
     61                     except ValueError:
     62                         pass
---> 63                 token.head.pos_ = corrected_pos
     64                 token.pos_ = corrected_pos
     65                 token.dep_ = 'dep'

token.pyx in spacy.tokens.token.Token.pos_.__set__()

KeyError: 'num'
```

When I feed  '2016.9 - 2017.' instead of '2016.9 - 2017.2', the error won't occur.
Also, there is no problem with '2017.2' .
I tried 'en' package and it worked with the sentence '2016.9 - 2017.2'.

**Comment by hiroshi-matsuda-rit** [comment]
@dwarfer7634 Thanks for reporting the error. I'm going to fix it.

**Comment by hiroshi-matsuda-rit** [comment]
@dwarfer7634 Thanks for reporting the problem. This had been fixed by v2.0.0. Please install the latest module and model, then load the model by executing `nlp = spacy.load('ja_ginza')`.

**Comment by dwarfer7634** [comment]
Thank you very much!
